client/scss/components/_icons.scss

```diff
@@ -168,8 +168,8 @@ svg.icon-spinner {
 
 // Media for Windows High Contrast mode
 
-@media (forced-colors: $media-forced-colours) {
-  .icon {
-    fill: $system-color-link-text;
-  }
-}
```

Comment:
Go ahead and delete this whole block. We only want to leave commented code in the codebase if we intend to reuse it at some point, which I don't think is true in this case.